components/threading/global_threads/thread_list/thread_list.scss

```diff
@@ -30,4 +36,12 @@
     .ThreadItem {
         border-bottom: var(--border);
     }
+
+    .tab-button-wrapper {
```

Comment:
this can be pulled to the first definition of `.tab-button-wrapper` and then nest the definition for `+ .tab-button-wrapper` inside it